django/conf/global_settings.py

```diff
@@ -543,6 +543,8 @@ def gettext_noop(s):
 CSRF_COOKIE_PATH = '/'
 CSRF_COOKIE_SECURE = False
 CSRF_COOKIE_HTTPONLY = False
+CSRF_USE_SESSIONS = False
+CSRF_SESSION_KEY = 'csrftoken'
```

Comment:
Yeah, screw that :D in the worst case make it an attribute on the CSRF class, so a user can override it if we go down the route with an extra class.